framework/CMakeLists.txt

```diff
@@ -59,15 +59,19 @@ endif ("${isSystemDir}" STREQUAL "-1")
 set (CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 
 set(MAPF_ROOT "/opt/multiap/framework")
+set(TMP_PATH "/tmp")
 
 if(TARGET_PLATFORM STREQUAL "linux")
     set(MAPF_ROOT "${CMAKE_INSTALL_PREFIX}")
     option(USE_USER_TMP_PATH "use /tmp/$USER instead of /tmp path" ON)
     if(USE_USER_TMP_PATH)
-	    set(USER "$ENV{USER}")
+        set(USER "$ENV{USER}")
+        set(TMP_PATH "/tmp/${USER}")
     endif()
+elseif (TARGET_PLATFORM STREQUAL "turris-omnia")
+    set(MAPF_ROOT "/opt/beerocks")
```

Comment:
same here - I generally prefer to keep the standard `/usr/bin` if it's not too much work. If it is, then perhaps `/opt/prplMesh` can be a good compromise.

Reply:
Changed to `/opt/prplmesh`